manager/allocator/networkallocator/networkallocator.go

```diff
@@ -878,3 +890,16 @@ func serviceNetworks(s *api.Service) []*api.NetworkAttachmentConfig {
 	}
 	return s.Spec.Task.Networks
 }
+
+//IsVipOnIngressNetwork check if the vip is in ingress network
+func (na *NetworkAllocator) IsVipOnIngressNetwork(vip *api.Endpoint_VirtualIP) bool {
```

Comment:
cosmetic: Could you change `Vip` to `VIP` and add a space between `//` and the start of the comment?